client/components/ConfigureActiveRuns/index.jsx

```diff
@@ -466,37 +468,46 @@ class ConfigureActiveRuns extends Component {
         const getTestVersions = function() {
             const testVersionOption = version => (
                 <option key={version.id} value={version.id}>
-                    {version.git_hash.slice(0, 7) +
+                    {moment(version.date).format('MMMM Do YYYY') +
                         ' - ' +
                         version.git_commit_msg.slice(0, 80) +
-                        '...'}
+                        '...' +
+                        ' - ' +
+                        version.git_hash.slice(0, 7) +
+                        ' - '}
```

Comment:
This formatting seems a little strange to me but it's a minor thing we can fix later if someone knows what to change it too